packages/containers/src/index.js

```diff
@@ -1,45 +1,208 @@
-import Action from './Action';
-import Actions from './Actions';
-import AppHeaderBar from './AppHeaderBar';
-import List from './List';
-import Notification from './Notification';
-import SidePanel from './SidePanel';
-
-// export all other components from react-talend-components
-
 import {
-	ActionDropdown,
+	// react-bootstrap proxified
+	Alert,
+	BootstrapBadge,
+	Breadcrumb,
+	BreadcrumbItem,
+	Button,
+	ButtonGroup,
+	ButtonToolbar,
+	Carousel,
+	CarouselItem,
+	Checkbox,
+	Clearfix,
+	ControlLabel,
+	Col,
+	Collapse,
+	Dropdown,
+	DropdownButton,
+	Fade,
+	Form,
+	FormControl,
+	FormGroup,
+	Glyphicon,
+	Grid,
+	HelpBlock,
+	Image,
+	InputGroup,
+	Jumbotron,
+	Label,
+	ListGroup,
+	ListGroupItem,
+	Media,
+	MenuItem,
+	Modal,
+	ModalBody,
+	ModalFooter,
+	ModalHeader,
+	ModalTitle,
+	Nav,
+	Navbar,
+	NavbarBrand,
+	NavDropdown,
+	NavItem,
+	Overlay,
+	OverlayTrigger,
+	PageHeader,
+	PageItem,
+	Pager,
+	Pagination,
+	Panel,
+	PanelGroup,
+	Popover,
+	ProgressBar,
+	Radio,
+	ResponsiveEmbed,
+	Row,
+	SafeAnchor,
+	SplitButton,
+	Tab,
+	TabContainer,
+	TabContent,
+	Table,
+	TabPane,
+	Tabs,
+	Thumbnail,
+	Tooltip,
+	Well,
+
+	// react-talend-components
+	// Action,
 	ActionBar,
+	ActionDropdown,
+	ActionSplitDropdown,
+	// Actions,
+	// AppHeaderBar,
 	Breadcrumbs,
 	CircularProgress,
+	CollapsiblePanel,
+	Dialog,
 	ConfirmDialog,
 	Drawer,
 	TreeView,
+	Badge,
 	Icon,
 	IconsProvider,
 	Layout,
+	// List,
+	// Notification,
+	Progress,
+	// SidePanel,
+	Status,
 	TooltipTrigger,
 	Typeahead,
 } from 'react-talend-components';
```

Comment:
https://gist.github.com/caridy/839d5359321604a12648#file-export-syntax-js-L29

please check a way to reduce # of lines on gist, however it's stage1 so idk if we can use it.